core/modules/wsched/SchedulerBase.h

```diff
@@ -105,6 +122,17 @@ class SchedulerBase : public wcontrol::Scheduler {
     int _priorityNext; ///< Priority to use starting with the next chunk.
 
     std::atomic<int> _inFlight{0}; //< Number of Tasks running.
+
+private:
+    /// The true purpose of _userQuerycount is to track how many different UserQuery's are on the queue.
+    /// Number of Tasks for each UserQuery in the queue.
+    std::map<uint64_t, int> _userQueryCounts;
```

Comment:
I don't understand the separation either. I'll move it to intTypes.h and see what happens.